WebRoot/js/zimbraMail/briefcase/view/ZmDetailListView.js

```diff
@@ -362,6 +362,39 @@ function(item) {
 	}
 };
 
+ZmDetailListView.prototype._getLabelFieldList =
+function () {
+	var headers = [];
+	headers.push(ZmItem.F_SELECTION);
+	headers.push(ZmItem.F_NAME);
+	headers.push(ZmItem.F_TYPE);
+	headers.push(ZmItem.F_SIZE);
+	headers.push(ZmItem.F_DATE);
+	headers.push(ZmItem.F_SHARES);
+	return headers;
```

Comment:
better write it this way?

return [
    ZmItem.F_SELECTION,
    ZmItem.F_NAME,
    ZmItem.F_TYPE,
    ZmItem.F_SIZE,
    ZmItem.F_DATE,
    ZmItem.F_SHARES
];

Reply:
Fixed.